{{ cookiecutter.package_name }}/.rtd-environment.yml

```diff
@@ -0,0 +1,8 @@
+name: rtd_sunpy
```

Comment:
Should this be templated by the package name?

Reply:
yeah it should